src/compas/utilities/functions.py

```diff
@@ -5,6 +5,9 @@
 
 __all__ = [
     'fibonacci',
+    'avrg',
```

Comment:
Better to move all these new functions into a specialized module `statistics` (and as a personal rant, `utilities` in general is such an annoyingly mixed bag of things I would eventually reconsider its existence as well, but that's beyond the scope of this PR).

Reply:
done